android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -397,6 +403,11 @@ private void setupPurchases(String apiKey, String appUserID,
                     verificationMode,
                     pendingTransactionsForPrepaidPlansEnabled,
                     automaticDeviceIdentifierCollectionEnabled);
+            
+            if (preferredUILocaleOverride != null) {
+                CommonKt.overridePreferredLocale(preferredUILocaleOverride);
```

Comment:
The [CommonKt.configure](https://github.com/RevenueCat/purchases-hybrid-common/blob/94794c632fe87b6f4d88f74e781b2b8c64d775fa/android/hybridcommon/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt#L947) accepts a nullable locale. It's preferred to pass that parameter at configure time, than setting it up later. Could you change that? 🙏